packages/database/scripts/setupNewDatabase.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$SCRIPT_DIR"
+cd ..
+
+if [ "$1" != "OK" ]; then
+  echo "Note: this script reads .env vars"
+  echo "   [1] Update DB_NAME etc... in database db.env"
+  echo ""
+  echo "re-run with OK to continue"
+  exit 1;
+fi
+
+DIR=$(pwd "$0")
+source "$DIR/../../scripts/bash/mergeEnvForDB.sh" 
+
+# Set default port in case it wasn't in .env
+: "${DB_PORT:=5432}"
+
+set -x
+
+IS_RDS=$(PGPASSWORD=$DB_PG_PASSWORD psql -qtAX -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "SELECT 1 FROM pg_roles WHERE rolname='rds_superuser'")
+
+PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "CREATE ROLE $DB_USER LOGIN PASSWORD '$DB_PASSWORD'" || echo "Role $DB_USER already exists?"
+if [ "$IS_RDS" = "1" ]; then
+  PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "GRANT rds_superuser TO $DB_USER"
+  PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "grant $DB_USER to postgres"
```

Comment:
```suggestion
  PGPASSWORD=$DB_PG_PASSWORD psql -h $DB_URL -p $DB_PORT -U $DB_PG_USER -c "GRANT $DB_USER to postgres"
```
Just for consistency